h/static/styles/annotations.scss

```diff
@@ -31,13 +38,38 @@
 .annotation-header,
 .annotation-footer {
   @include pie-clearfix;
-  margin: .8em 0;
 }
 
-.annotation-header { margin-top: 0 }
+.annotation-header {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
+
+.annotation-header__share-info {
+  color: $color-gray;
+}
+
+.annotation-header__group {
+  color: $color-gray;
+  font-size: $body1-font-size;
+}
+
+.annotation-header__group-name {
+  display: inline-block;
+  margin-left: 5px;
+}
+
 .annotation-footer { margin-bottom: 0 }
 
+.u-flex-spacer {
+  flex-grow: 1;
+}
+
 .annotation-section {
+  margin-top: 14px;
+  margin-bottom: 14px;
```

Comment:
This is problematic for replies, which have targets but no selectors (q.v. `annotation.html`):

<img width="423" alt="screen shot 2015-10-27 at 16 04 37" src="https://cloud.githubusercontent.com/assets/3602/10764081/72d3acf8-7cc4-11e5-9330-b0b6df1839e5.png">